Application/components/Server/MemberList.tsx

```diff
@@ -191,17 +192,18 @@ export function MemberList({isAdmin, chatID}: any) {
       </Text>
       {membersList.map((member, index) => (
         <div>
+          {/* <Text>{member}</Text> */}
           <Menu shadow="md" position="left" width={225} withArrow >
               <Menu.Target>
-                  <Button fullWidth variant="gradient">
+                  <Button style={{width: "215px"}} variant="gradient">
```

Comment:
Don't hardcode the size. Just specify `grow` as a field and the button will grow to its max width within its parent container.